cluster/gce/config-default.sh

```diff
@@ -289,8 +289,10 @@ ENABLE_RESCHEDULER="${KUBE_ENABLE_RESCHEDULER:-true}"
 ENABLE_IP_ALIASES=${KUBE_GCE_ENABLE_IP_ALIASES:-false}
 NODE_IPAM_MODE=${KUBE_GCE_NODE_IPAM_MODE:-RangeAllocator}
 if [ ${ENABLE_IP_ALIASES} = true ]; then
-  # Size of ranges allocated to each node. Currently supports only /32 and /24.
```

Comment:
> Currently supports only /32 and /24.

Sorry if this is known, has this limitation been lifted yet? I'm still seeing it on https://cloud.google.com/vpc/docs/alias-ip. @satyasm

Reply:
Confirmed this is not an issue anymore. Thanks @satyasm for verifying!